analyzers/tests/SonarAnalyzer.UnitTest/Rules/Hotspots/CommandPathTest.cs

```diff
@@ -27,21 +27,37 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class CommandPathTest
     {
+        private readonly VerifierBuilder builderCS = new VerifierBuilder().WithBasePath("Hotspots").AddAnalyzer(() => new CS.CommandPath(AnalyzerConfiguration.AlwaysEnabled));
+        private readonly VerifierBuilder builderVB = new VerifierBuilder().WithBasePath("Hotspots").AddAnalyzer(() => new VB.CommandPath(AnalyzerConfiguration.AlwaysEnabled));
+
         [TestMethod]
         public void CommandPath_CS() =>
-            OldVerifier.VerifyAnalyzer(@"TestCases\Hotspots\CommandPath.cs", new CS.CommandPath(AnalyzerConfiguration.AlwaysEnabled), MetadataReferenceFacade.SystemDiagnosticsProcess);
+            builderCS.AddPaths("CommandPath.cs")
+                .AddReferences(MetadataReferenceFacade.SystemDiagnosticsProcess)
```

Comment:
If you want, you can add this dependency to the top builders to avoid duplication, and then each test could be on the same line. Up to you.